src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,61 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+
+    @Test
+    public void testStringSplitByDelimiter() {
+        String parameter = "1,2";
+        String delimiter = ",";
+
+        String[] result = parameter.split(delimiter);
```

Comment:
```suggestion
        String[] result = "1,2".split(",");
```

굳이 로컬 변수로 빼지 않고 위와 같이 구현하는 것은 어떨까?